tests/test_fixtures/rule_based_profiler/example_notebooks/BasicExample_RBP_Instantiation_and_running.ipynb

```diff
@@ -0,0 +1,1778 @@
+     ]
+    }
+   ],
+   "source": [
+    "print(parameter_container.parameter_nodes)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 55,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'%Y-%m-%d %H:%M:%S'"
+      ]
+     },
+     "execution_count": 55,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "parameter_container.parameter_nodes[\"parameter\"][\"parameter\"][\"my_value_set\"][\"value\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The result contains our matching `datetime` pattern, which is `'%Y-%m-%d %H:%M:%S'`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### `NumericMetricRangeMultiBatchParameterBuilder`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `NumericMetricRangeMultiBatchParameterBuilder` is able to provide range estimations across batches using sampling methods. For instance, if we expect a table's row_count to change between batches, we could calculate the min / max values of row_count by using the `NumericMetricRangeMultiBatchParameterBuilder`. These parameters could then be used `Expectations` take in ranges, like `ExpectTableRowCountToBeBetween`, or `ExpectColumnValuesToBeBetween`.\n",
+    "\n",
+    "In this example, we will be taking a single metric, `column.mean` and calculating it for a single column, `total_amount`. The parameter we will be building is the column mean-range, which is the min-max values of the `total_amount` column across random samples of 12 batches of the 2018 `taxi_data` dataaset. \n",
+    "\n",
+    "We will also be passing in specifications for sampling techniques, namely `bootstrap` sampling with a false-positive rage of less than 0.01. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 58,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "domain: Domain = Domain(domain_type=MetricDomainTypes.COLUMN, domain_kwargs = {'column': 'total_amount'})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 59,
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "TypeError",
+     "evalue": "__init__() got an unexpected keyword argument 'sampling_method'",
```

Comment:
Is this a known issue?

Reply:
Oh no, this was something that wasn't saved... the parameter name was changed in a recent commit.